test/functional/fixtures/api/es-next/selector/test.js

```diff
@@ -271,6 +275,51 @@ describe('[API] Selector', function () {
                     });
             }
         );
+
+        it('Should raise error if addCustomMethods method argument is not object',
```

Comment:
We use this approach everywhere, e.g. https://github.com/DevExpress/testcafe/blob/master/test/functional/fixtures/api/es-next/client-function/testcafe-fixtures/client-fn-test.js#L32